run.sh

```diff
@@ -28,9 +28,8 @@ $DOCKER_REQUIRE_SUDO docker run --rm -it --privileged \
   -e HOME="$HOME" \
   -e USER_NAME="$USER" \
   -e USER_UID=$USER_UID \
-	-e USER_GID=$USER_GID \
+  -e USER_GID=$USER_GID \
   -w "$(abspath `pwd`)" \
-  --privileged \
```

Comment:
I don't know if it's required, but I doubt it's required twice. (Look at the file a few lines above the diff context.)